app/src/main/java/com/endava/myendava/models/Tag.java

```diff
@@ -99,4 +104,8 @@ public String toString() {
                 ", tagDescription='" + tagDescription + '\'' +
                 '}';
     }
+
+    public boolean isEditTag(){
```

Comment:
why have a method that always returns false? 🤔

Reply:
Sorry that method is never used, I forgot to delete id